crates/typst/src/layout/place.rs

```diff
@@ -1,6 +1,8 @@
+use typst_macros::scope;
+
 use crate::diag::{bail, At, Hint, SourceResult};
 use crate::engine::Engine;
-use crate::foundations::{elem, Content, Packed, Smart, StyleChain};
+use crate::foundations::{elem, Content, Packed, Smart, StyleChain, Unlabellable};
```

Comment:
rust-analyzer always imports form `typst_macros`, but `scope` is supposed to be imported from `foundations` (where it is reexported)